react/Section/Section.js

```diff
@@ -47,5 +47,7 @@ Section.defaultProps = {
   className: '',
   header: false,
   pullout: false,
-  slim: false
+  slim: false,
+  tone: '',
```

Comment:
But there's no empty string declared as a proptype? 🤔

Reply:
I think it would be nice if we group all the defaults in one place 
https://github.com/seek-oss/seek-style-guide/pull/552/files#diff-04146d12928269aed074c3da275eed92R21
https://github.com/seek-oss/seek-style-guide/pull/552/files#diff-04146d12928269aed074c3da275eed92R47

Reply:
No 🤔... flow doesn't seem to care about this

Reply:
I tried removing defaulting, seems flow doesn't care at all

Reply:
React warning do not care at all 🤔 Any thoughts on that?

As a short summary, I feel this is not ideal to default to an empty string because it does not mean anything from the component point of view

Reply:
I think from the component point of view, you don't need to pass in any of these props because a default is provided for you. I wouldn't want to be warned about `tone` here because it's valid to not provide one. Most apps don't use the `tone` prop with `Section`

Reply:
if it were proptypes what would you put there. `tone: PropTypes.string.isRequired` or `tone: PropTypes.string`?

Reply:
I think it should be `tone: PropTypes.string.isRequired` because the Section component requires a tone prop. The default props provide one if it isn't passed in. I realise this isn't how prop types have been used throughout the styleguide but I think this is more correct. 

Our flow types are describing the component and stating which props is requires. Tone is required for the component to render in this case so I think the type should mark tone as required.